src/main/java/com/mongodb/kafka/connect/util/jmx/SourceTaskStatisticsMBean.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.kafka.connect.util.jmx;
+
+/**
+ * <p>A standard MBean interface for a {@link com.mongodb.kafka.connect.source.MongoSourceTask}.</p>
+ *
+ * <p>This interface is NOT part of the public API.  Be prepared for non-binary compatible changes in minor releases.</p>
+ *
+ * @since TODO
+ */
+public interface SourceTaskStatisticsMBean {
+
+  /** @return Milliseconds spent in the task, including any sub-phases. */
```

Comment:
All three of these times could (and probably should) be represented as well, e.g.

* taskPollTimeMsLessThanX/Y/Z/... in powers of 10
* readTimeMsLessThanX/Y/Z...
* taskBetweenPollMsLessThanX/Y/Z/... 

and consider renaming these to emphasize that they are totals

Reply:
Defer to subsequent PR